src/Sylius/Behat/Context/Api/Shop/AddressContext.php

```diff
@@ -163,6 +169,14 @@ public function iSetTheAddressOfAsDefault(string $fullName): void
         $this->customerClient->update();
     }
 
+    /**
+     * @When /^I try to view details of (address belonging to "([^"]+)")$/
+     */
+    public function iTryToViewDetailsOfAddressBelongingTo(AddressInterface $address): void
+    {
+        $this->client->showByIri($this->iriConverter->getIriFromItem($address));
```

Comment:
I guess it should be:
```suggestion
        $this->addressClient->showByIri($this->iriConverter->getIriFromItem($address));
```